### PR TITLE
Enable chapters labels numbers units toc 

### PR DESCRIPTION
**Pull request by ajluk**
Issue: #1782

### CODE REVIEW

**Comment by ajluk** [comment]
How it looks now: 
image

**Comment by ajluk** [comment]
@helenemccarron and @brittany-johnson I have added you to this review, because I wanted to confirm it is the right solution for adding units labels and numbers via recipe. I was modeling my changes on this PR by Thomas: https://github.com/openstax/cnx-recipes/commit/7804a5b1b57321f51151182b05789531290ac3b7#diff-953359d42ecb407753858da439153886.

File: recipes/books/_common-stuff.scss [hunk 1]
```diff
@@ -13,9 +13,17 @@ $Config_TocTitleWord: "Contents";
...
 
+$Config_PartType_Unit_TitleContent   : (os-number: counter(unit), os-divider: ' ' );
 $Config_PartType_Chapter_TitleContent   : (os-number: counter(chapter), os-divider: ' ' );
 $Config_PartType_Appendix_TitleContent  : (os-number: counter(appendix, upper-alpha), os-divider: ' | ');
 
+
+@if TEMPLATE() == "TEMPLATE1" {
+$Config_PartType_Chapter_TitleContent   : (os-number: "Chapter " counter(chapter), os-divider: ' ' );
+$Config_PartType_Unit_TitleContent   : (os-number:"Unit " counter(unit), os-divider: ' ' );
+$Config_PartType_Appendix_TitleContent  : (os-number: "Appendix " counter(appendix, upper-alpha), os-divider: ' ');
```
Comment (helenemccarron) [inline recipes/books/_common-stuff.scss]: I am not sure the issue calls for the appendices to be modified, maybe check with @lwtchu ?

Comment (wilkus27) [reply recipes/books/_common-stuff.scss]: Please, ask also about divider. Should it be ` ' '` or `' | '`?

Comment (ajluk) [reply recipes/books/_common-stuff.scss]: @helenemccarron and @wilkus27 I have confirmed with Larissa today that the Appendices labels should be added within this PR as well and no divider is needed the (the space is already there).

Comment (ajluk) [reply recipes/books/_common-stuff.scss]: @helenemccarron and @wilkus27 I have confirmed with Larissa today that the Appendices labels should be added within this PR as well and no divider is needed the (the space is already there).

**Pull request merged**